src/Services/CustomCommandService.cs

```diff
@@ -31,20 +31,23 @@ public CustomCommandService()
         public IReadOnlyCollection<CustomCommand> GetCommands()
             => _commands.AsReadOnly();
 
-        public bool TryGetCommand(string name, out CustomCommand command)
+        public bool TryGetCommand(string name, string[] args, out CustomCommand command)
         {
-            command = GetCommandByName(name);
+            command = GetCommandByName(name, args);
+
             return command != null;
         }
 
-        public CustomCommand GetCommandByName(string name)
-            => _commands.FirstOrDefault(c => c.Matches(name));
-        public bool ContainsCommand(string name)
-            => _commands.Any(c => c.Matches(name));
+        public CustomCommand GetCommandByName(string name, string[] args)
```

Comment:
We should probably allow commands to be fetched by only their name. The arguments don't really make sense in there I feel.

Reply:
Added overload method